content/price.json

```diff
@@ -12,18 +12,18 @@
       "title": "Fixed",
       "info": "Set your price for accessing this data set. The datatoken for this data set will be worth the entered amount of OCEAN.",
       "tooltips": {
-        "communityFee": "Explain community fee...",
-        "marketplaceFee": "Explain marketplace fee..."
+        "communityFee": "This fees go to Ocean DAO for teams to improve the tools, build apps, do outreach, and more. A small fraction also goes to burning OCEAN. These fees are collected in the action of consuming a data asset.",
```

Comment:
```suggestion
        "communityFee": "Goes to [Ocean DAO](https://oceanprotocol.com/dao) for teams to improve the tools, build apps, do outreach, and more. A small fraction is used to burn OCEAN. This fee is collected when downloading or using an asset in a compute job.",
```

Reply:
At first, I wrote it with the link to Ocean DAO but I removed it [here](https://github.com/oceanprotocol/market/pull/771/commits/fed6297f23b5a54962d6649e0b0ee5163b8fd38d) because the link doesn't work inside the tooltip.

Reply:
I implemented your other changes

Reply:
ah cool, was not sure if our component handles that, guess it should but that's outside of scope of this issue